include/api/CInferenceModelDefinition.h

```diff
@@ -112,6 +112,27 @@ class API_EXPORT CLogisticRegression final : public CAggregateOutput {
     TDoubleVec m_Weights;
 };
 
+//! Allows to use logistic regression aggregation.
+//!
+//! Given a weights vector $\vec{w}$ as a parameter and an output vector from the ensemble $\vec{x}$,
+//! it computes the logistic regression function \f$1/(1 + \exp(-\vec{w}^T \vec{x}))\f$.
```

Comment:
This comment looks wrong. The forest is predicting `sum_i{ leaf_i } = log(1 + x)`, so for the weighted case the aggregator would be `exp(w^t x) - 1` no?

Reply:
In fact, thinking about it, 1 is the offset is a parameter which we also need to supply.

Reply:
You are right, the aggregator computes `exp(w^t x)` cf. `CMsle::transform`.

Reply:
Ok great.

and the comment on the offset was me forgetting what we're actually doing. For posterity, we compute leaf values by `argmin_x (log(offset + exp(prediction)) - log(offset + actual))^2`, i.e. the important part is that prediction is exponentiated so the forest is effectively predicting logs. The offset just governs how we penalise relative errors between the prediction actuals at the scale of offset.